cwizard2011/maintenace-tracker-app
# #157573260 Fixed mobile responsiveness on Landing page

#### What does this PR do?
Fix responsiveness on the landing page

#### How should this be manually tested?
clone this repo, navigate to the index.html file. check the responsiveness with chrome developer tool. You can also check on any device by visiting this link https://cwizard2011.github.io/maintenace-tracker-app/UI/

#### What are the relevant pivotal tracker stories?
#157573260
